### DISCUSSION TITLE
Problem with submitting forms to nodejs server

### DISCUSSION THREAD

**Discussion by van-vince** [refinedev/refine#4890] [Q&A] [answered]
Please am having problem trying to use useModalForm by following the example in documentation. https://codesandbox.io/p/sandbox/github/refinedev/refine/tree/next/examples/form-material-ui-use-drawer-form?embed=1&file=/src/App.tsx:22,39
It works perfectly with fakeApi but when I connect by NodeJs server am not able to create user. upon inspecting the error I get 404 not found. But i realised that the post request hit http://localhost:8000/api/v2/user instead of http://localhost:8000/api/v2/user/create-user. even though i have set create in resource. 
resources={[
        {
            name: "user",
            create: "user/create-user",
            list: "/users",
        },
I am finding it difficult to resolve this. any help will be appreciated. server works perfectly with postman
below are some solutions i have tried..
I added am meta to refinecoreProps and swizzled dataProvider to edit the create url
![Screenshot 2023-08-30 at 9 26 25 AM](https://github.com/refinedev/refine/assets/67305320/36c96691-a7e9-4ab6-8935-85cda741bb10)
![Screenshot 2023-08-30 at 9 27 03 AM](https://github.com/refinedev/refine/assets/67305320/6a7a04dc-2e13-42d7-8715-0d4255c7feff)
![Screenshot 2023-08-30 at 9 37 00 AM](https://github.com/refinedev/refine/assets/67305320/dcd52798-35a9-4e22-91f7-ef08bac98d75)
I  change the server url for create user to http://localhost:8000/api/v2/user and it was successful. so is there any other way to make the request point to http://localhost:8000/api/v2/user/create-user.

**Comment by BatuhanW** [comment, marked as answer]
Hey @van-vince, did you update Refine component's `dataProvider` prop to pass swizzled data provider?

**Reply by van-vince** [reply]
Thank you. It worked after properly importing dataProvider from swizzled dataProvider. Initially I taught it will automatically override the default dataProvider.
